tests/resources/valid/control_flow/if_check.py

```diff
@@ -1,36 +1,40 @@
 b = 40
 
-if a:
-    print("hello world")
+if True:
+    print(f"hello world")
 
-if c:
-    d = 4
+if False:
+    f"hello"
 else:
-    asdf
+    f"world"
+
+cond = True and False
+cond = True or False
 
 if cond:
-    other
-    print('hello \"world\"')
+    f"asdf"
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Statement seems to have no effect](https://app.codacy.com/app/JSAbrahams/mamba/pullRequest?prid=4098440)